README.md

```diff
@@ -94,6 +94,34 @@ app.use(i18n());
 
 **Middleware dependency** : Language middleware
 
+### IP Filter
+
+Accept or reject requests based upon their IP address.
+Controlled by 2 new environment variables
+
+FILTER_IP = true or false (turn it on or off)
+PROTECTED_IPS = SPACE separated list of (a) individual IP addresses (b) IP address ranges
```

Comment:
"PROTECTED_IPS" is not 100% self explanatory for me. Will we deny access or allow access ? If I'm not sure right now, I won't be sure at 2 AM after receiving the text